ClinGen/clincoded
# Connects to #1506. Connects to #1507. Connects to #1508. Connects to #1551. Connects to #1498.

**Technical notes:**
The implementation of affiliation affects entire workflows in both of the GCI and VCI. As a result, testing should be done throughout the entire application.

**Features and functionality to be tested:**
1. Signed as a member of an affiliation or as yourself (without affiliation).
2. The affiliation utility bar in the header upon signing s a member of an affiliation.
3. Dashboard page with the associated variant interpretations and gene-disease records when signed in either as a member of an affiliation or as yourself (without affiliation).
4. The ability to **edit** evidence and scores associated with the affiliation you signed as a member of within any given gene-disease record.
5. The ability to **view** the gene-disease record's classification matrix associated with your affiliation.
6. The ability to **view** and **edit** the gene-disease record's classification associated with your affiliation.
7. The ability to **view**  the gene-disease record's evidence summary associated with your affiliation.
8. The ability to **view** and **curate** the gene-disease record's variant associated with your affiliation.
9. The ability to **edit** the variant interpretations associated with your affiliation.
10. The ability of **view** and **edit** the criteria evaluations in the variant interpretation associated with your affiliation.
11. The ability to **view** and **edit** the evidence (PMID) in the variant interpretation associated with your affiliation.
12. The ability to **view** the evaluation summary for the variant interpretation associated with your affiliation.
13. The ability to **edit** the pathogenicity classification in the evaluation summary for the variant interpretation associated with your affiliation.
14. Dashboard page with historical items representing your curation activities while signed in as a member of a given affiliation.

**Steps to test #1551:**
1. Go to an existing gene-disease record page (aka Curation-Central).
2. Expect to see the **Preview Evidence Summary** button in the header.
3. Click on the **Preview Evidence Summary** button and expect to see a new window/tab showing the evidence summary with **PREVIEW** watermarks across the page.

**Steps to test #1498:**
1. While on the evidence summary page on a test/demo instance, expect to see the red demo notice bar at the top of the page.
